.azure/templates/jobs/build/test_strimzi.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       - bash: "mvn -e -V -B -Dmaven.javadoc.skip=true -Dsurefire.rerunFailingTestsCount=5 -Dfailsafe.rerunFailingTestsCount=2 install"
         env:
           # Test container optimization and also we eliminate pulling RYUK image from DockerHub to prevent limits
-          TESTCONTAINERS_RYUK_DISABLED: TRUE
+          TESTCONTAINERS_RYUK_DISABLED: FALSE
```

Comment:
Why is RYUK now enabled? :).

Reply:
RYUK is doing some cleanup of the containers and networks. I originally enabled it before we had the shared network support. Let me see if it is still needed or not.

Reply:
Sure, thanks.

Reply:
@see-quick Looks like it works, so let's keep RYUK disabled.